src/components/RankingList.tsx

```diff
@@ -4,18 +4,19 @@ import Divider from "@/components/Divider";
 import { useState } from "react";
 import type { ThemeType } from "@/types/ThemeType";
 import { useTheme } from "@emotion/react";
-import type { RankingItemType } from "@/types/RankingItemType";
 
-const rankingListItemViewCount = 6;
+const RANKING_LIST_ITEM_VIEW_COUNT = 6;
 
 const RankingList = () => {
-  const [viewCount, setViewCount] = useState(rankingListItemViewCount);
+  const [viewCount, setViewCount] = useState(RANKING_LIST_ITEM_VIEW_COUNT);
   const theme = useTheme();
-  const rankingItem: RankingItemType[] = rankingItemMock;
+  const isClosed = () => {
+    return viewCount === RANKING_LIST_ITEM_VIEW_COUNT;
+  };
```

Comment:
```suggestion
  const isClosed = viewCount === RANKING_LIST_ITEM_VIEW_COUNT;
```

함수대신 변수로 사용할 수 있어보여요 😊